Signal/src/Models/OWSCall.h

```diff
@@ -2,60 +2,58 @@
 //  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
```

Comment:
The "message adapters" including `OWSCall` have been replaced by `ConversationViewItem`.